tests/ttnn/unit_tests/test_reshape.py

```diff
@@ -304,6 +309,26 @@ def test_reshape_tile_with_padding(input_shape, output_shape, layout, device):
     ttnn_output = ttnn.reshape(input_tensor, output_shape)
     assert layout == ttnn_output.layout
     output = ttnn.to_torch(ttnn_output)
+    assert_with_pcc(torch_result, output, 0.9999)
+
+
+# issue 15048
+def test_broken_reshape(device):
```

Comment:
this works now, right? "test_broken_reshape" is such a confusing name :D

Reply:
I just copied the code from the issue, I am going to rename the test to test_previously_failing_test